src/Tickets/Flexible_Tickets/template-tags.php

```diff
@@ -165,4 +165,58 @@ function tec_tickets_get_default_ticket_type_label_lowercase( string $context =
 			$context
 		);
 	}
-}
\ No newline at end of file
+}
+
+if ( ! function_exists( 'tec_tickets_get_default_ticket_type_label_plural' ) ) {
+	/**
+	 * Returns the filtered default Ticket Type label in plural.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $context The context in which this string is filtered, e.g. 'verb' or 'template.php'.
+	 *
+	 * @return string The plural version of default ticket type label.
+	 */
+	function tec_tickets_get_default_ticket_type_label_plural( string $context = '' ): string {
+		/**
+		 * Allows customization of the default ticket type label plural.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $label   The default ticket type label, defaults to "Single Tickets".
+		 * @param string $context The context in which this string is filtered, e.g. 'verb' or 'template.php'.
+		 */
+		return apply_filters(
+			'tec_tickets_get_default_ticket_type_label_plural',
+			_x( 'Single Tickets', 'default ticket type label in plural', 'event-tickets' ),
+			$context
+		);
+	}
+}
+
+if ( ! function_exists( 'tec_tickets_get_default_ticket_type_label_plural_lowercase' ) ) {
+	/**
+	 * Returns the filtered default Ticket Type label in plural and lowercase.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $context The context in which this string is filtered, e.g. 'verb' or 'template.php'.
+	 *
+	 * @return string The plural and lowercase version of default ticket type label.
+	 */
+	function tec_tickets_get_default_ticket_type_label_plural_lowercase( string $context = '' ): string {
```

Comment:
Same question for this template tag: should this be here or in the more general ET template tags?

Reply:
Good call, [moved](https://github.com/the-events-calendar/event-tickets/pull/2875/commits/f27d4f7224a461dd472563f05eb5383bcfaeaa27) them to base template tags for tickets: